pkg/arvo/sys/zuse.hoon

```diff
@@ -1134,6 +1135,7 @@
           {$talk p/tank}                                ::
           {$text p/tape}                                ::
           {$veer p/@ta q/path r/@t}                     ::  install vane
+          [%view =duct]                                 ::  watch session blits
```

Comment:
I'm not a fan of the ducts in these cards. It's not a pattern we've used elsewhere, it (at least arguable) conflicts with the purpose of ducts. It's a kind of fake, forward-looking modularity.

The design of %dill's namespace (`/dx/sessions//line`) suggests the intention of atomic session ids. This too is a kind of fake, forward-looking modularity, conflicting with the preceding instance, but arguably more defensible (what else would you do)?

I think it probably makes sense to reconcile these things now, and replace these ducts with atomic identifiers (`@tas` or `@ta`). We could use `%$` for the default session, as the new namespace does, and crash on any other identifiers until sessions are made real. Or, we could defer the session mechanism entirely, embracing The One Terminal and just send `[%view ~]` and `[%flee ~]` (ironically enough, the same noun).

Reply:
Yeah, my plan here was for this to use ducts only as a means to get webterm working, and then to transition to using more formal session ids immediately afterward. This is indeed what the `/dx/sessions//line` hints at. iow the fakeness and forward-looking-ness are intentional.

I was gonna say, this is more a matter of whether or not to block the PR on this, but it seems that as long as I don't touch the runtime-facing interface, we can become 80% real with 20% of the work. I'll try and push on this a little harder. (:

Reply:
Real sessions will be awesome, but I don't mean to push you to do that now.

In case it wasn't clear, the suggestion of `[%view ~]` and `[%flee ~]` is simply for %dill to automatically use the duct of the single, axiomatically available terminal (a la `[%flog ...]`).

Reply:
It's a good suggestion! Implemented as of the push just now.